src/main/java/com/example/codeE/helper/ExcelHelper.java

```diff
@@ -1,26 +1,59 @@
-package com.example.codeE.util.excelHelper;
-
-import org.apache.poi.ss.usermodel.Workbook;
-import org.apache.poi.ss.usermodel.WorkbookFactory;
-import org.apache.poi.ss.usermodel.Sheet;
-import org.apache.poi.ss.usermodel.Row;
-import org.apache.poi.ss.usermodel.Cell;
-import org.springframework.web.multipart.MultipartFile;
+package com.example.codeE.helper;
 
+import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.lang.reflect.Field;
-import java.util.Objects;
-import java.util.List;
 import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
 import java.util.UUID;
 
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.WorkbookFactory;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.example.codeE.constant.Constant;
+
 public class ExcelHelper {
-    public static boolean isValidExcelFile(MultipartFile file){
+        public static boolean isValidExcelFile(MultipartFile file){
         return Objects.equals(file.getContentType(), "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" );
     }
+     public static <T> void writeExcel(XSSFWorkbook workbook, List<T> dataList, String fileName, String sheetName){
+        try {
+            Sheet sheet = workbook.createSheet(sheetName);        
+            Row headerRow = sheet.createRow(0);
+            int cellIndex = 0;
+            for (java.lang.reflect.Field field : dataList.get(0).getClass().getDeclaredFields()) {
+                Cell cell = headerRow.createCell(cellIndex++);
+                cell.setCellValue(field.getName());
+            }
 
+            int rowIndex = 1;
+            for (Object data : dataList) {
+                Row row = sheet.createRow(rowIndex++);
+                cellIndex = 0;
+                for (java.lang.reflect.Field field : data.getClass().getDeclaredFields()) {
+                    field.setAccessible(true);
+                    Cell cell = row.createCell(cellIndex++);
+                    cell.setCellValue(String.valueOf(field.get(data)));
```

Comment:
for example, if we export users, do we also export the hash pass in the excel file too? 
is that ok to show hash pass or any sensitive information?

Reply:
Why do we export password? It is meaningless.